src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-directories/cephfs-directories.component.html

```diff
@@ -30,7 +30,8 @@
       </div>
       <div class="card-body">
         <ng-container *ngIf="selectedDir.path !== '/'">
-          <legend i18n>Quotas</legend>
+          <legend class="float-none"
+                  i18n>Quotas</legend>
```

Comment:
since the `legend` is something that is used in many other places and this bug could be possibly lurking around in other areas, what about overriding the `legend` float: none from the styles.scss with a comment?

Reply:
Looks like bootstrap kind of upgraded the [legend styling in bootstrap 5](https://getbootstrap.com/docs/5.0/content/reboot/#forms), and I think its right around that time we started seeing these issues in the UI